docs/runtime/transfer_learning.md

```diff
@@ -0,0 +1,199 @@
+                                                            image_size=IMG_SIZE)
+
+current_dir = pathlib.Path(__file__).parent
+with open(current_dir/'cats_and_dogs.input.bin', 'wb') as in_stream:
+    with open(current_dir/'cats_and_dogs.output.bin', 'wb') as out_stream:
+        for image,label in train_dataset:
+            # print(image.numpy().dtype)
+            in_stream.write(image.numpy().astype(np.float32).tobytes())
+            # print(label)
+            out_stream.write(label.numpy().astype(np.float32).tobytes())
+```
+
+## Feature extraction
+The first step to align the model to our problem can be inserting customs ops on the top of the model. The base of the model (with unchanged weights) remains the same.
+
+```python
+import tensorflow as tf
+import numpy as np
+
+BATCH_SIZE = 32
+IMG_SIZE = (160, 160)
+IMG_SHAPE = IMG_SIZE + (3,)
+
+base_model = tf.keras.applications.MobileNetV2(input_shape=IMG_SHAPE,
+                                               include_top=False,
+                                               weights='imagenet')
+
+base_model.trainable = False
+image_batch = tf.zeros([32, 160, 160, 3])
+feature_batch = base_model(image_batch)
+
+# feature extraction
+global_average_layer = tf.keras.layers.GlobalAveragePooling2D()
+feature_batch_average = global_average_layer(feature_batch)
+prediction_layer = tf.keras.layers.Dense(1, activation='relu')
+
+# applying preprocessing
+preprocess_input = tf.keras.applications.mobilenet_v2.preprocess_input
+
+inputs = tf.keras.Input(shape=(160, 160, 3))
+x = preprocess_input(inputs)
+x = base_model(x, training=False)
+
+# add additional layers on the top
+x = global_average_layer(x)
+outputs = prediction_layer(x)
+model = tf.keras.Model(inputs, outputs)
+
+# Convert to tflite
+converter = tf.lite.TFLiteConverter.from_keras_model(model)
+tflite_model = converter.convert()
+
+# Save tflite model model.
+with open('customized_mobilenetv2.tflite', 'wb') as f:
+  f.write(tflite_model)
+```
+For more information about feature extraction and fine tuning, please refer to [transfer learning tutorial](https://www.tensorflow.org/tutorials/images/transfer_learning).
+
+## Convert tflite model to circle model
+In the moment of writing, the `circle+` format is developed so it will the other possibility to handle model training.
+Note that `one-import` can be found in the installaion directory (determined by `-DCMAKE_INSTALL_PREFIX` cmake flag which can be added to `./nnfw configure` command).
+```bash
+one-import tflite -i customized_mobilenetv2.tflite -o customized_mobilenetv2.circle
+```
+
+## Run training
+Let's train the model using `onert_train`. In this case we want to test only 2 last layers: `Mean` and `FullyConnected` (created from `GlobalAveragePooling2D` and `Dense` in TensorFlow) Note that the other possibility is using `circle+` capabilities.
+NOTE TO REMOVE: Due to problems with exporing, the training was launched on https://github.com/Samsung/ONE/pull/13226
```

Comment:
```suggestion
NOTE TO REMOVE: Due to problems with exporting, the training was launched on https://github.com/Samsung/ONE/pull/13226
```

Reply:
right, it's great that you catch it, because line line `NOTE TO REMOVE: ...` is no more valid since the problem resolved on master